parm/gdas/staging/atm_lgetkf_bkg.yaml.j2

```diff
@@ -0,0 +1,30 @@
+{% set ftype_list = ['fv_core.res', 'fv_srf_wnd.res', 'fv_tracer.res', 'phy_data', 'sfc_data'] %}
+{% set time_list = [current_cycle] %}
+
+mkdir:
+{% for imem in range(1,NMEM_ENS+1) %}
+{% set memchar = 'mem%03d' | format(imem) %}
+- '{{ DATA }}/bkg/{{ memchar }}'
+- '{{ DATA }}/anl/{{ memchar }}'
+{% endfor %}
+copy:
+{% for time in time_list %}
+{% set prefix = time | to_fv3time %}
+{% for imem in range(1,NMEM_ENS+1) %}
+{% set memchar = 'mem%03d' | format(imem) %}
+{% set src_dir = COM_ATMOS_RESTART_TMPL | replace('${ROTDIR}', ROTDIR)
+                                        | replace('${RUN}', 'enkfgdas')
+                                        | replace('${YMD}', previous_cycle | to_YMD)
+                                        | replace('${HH}', previous_cycle | strftime('%H'))
+                                        | replace('${MEMDIR}', memchar) %}
```

Comment:
Using the built-in filter `replace` and chaining them is nice trick!

Can we write a custom filter `replace_tmpl` in `wxflow`:
```jinja2
replace_tmpl({'ROTDIR': ROTDIR, 'RUN': 'enkfgdas', 'YMD': previous_cycle | to_YMD})
```
such that `replace_tmpl` replaces the key `${key}` with `value`?

Reply:
Yeah, that seems useful and simple enough to implement. I'll give it a try